core/ide/src/main/java/org/alice/stageide/oneshot/OneShotUtilities.java

```diff
@@ -75,10 +76,10 @@ public static List<StandardMenuItemPrepModel> createMenuItemPrepModels(InstanceF
       ThisFieldAccessFactory thisFieldAccessFactory = (ThisFieldAccessFactory) instanceFactory;
       UserField field = thisFieldAccessFactory.getField();
       models.add(RenameFieldComposite.getInstance(field).getLaunchOperation().getMenuItemPrepModel());
+      AbstractType<?, ?, ?> fieldType = field.getValueType();
+      boolean isNotTargetingCamera = !fieldType.isAssignableTo(SCamera.class) && !fieldType.isAssignableTo(SVRUser.class);
```

Comment:
isNotTargetingCamera can be final

Reply:
It could, but it would be one of very few (any?) final local variables in our code base, including 3 previous variables in this method that could be final.
Between the verbosity and the limited scope of a local variable, I am disinclined to use final for locals. I am going to mull further and let this sit for another day.

I think Java picked the wrong default. We should have to declare that a variable can be changed.